rdkcentral/rdkservices
# XIONE-1313 : Add DeviceIdentification/DisplayInfo implementation for …

…Realtek

Reason for change:
* Add DeviceIdentification/DisplayInfo implementation for Realtek SOC
Test Procedure:
* Add DeviceIdentification, DisplayInfo in PACKAGECONFIG in rdkservices_git.bbappend for Realtek SOC layer
  and build RDKServices, DeviceIdentification, DisplayInfo plug-in should be build and works.
Risks: Low

Change-Id: I4708e00df16de74617168f099b7f8021b1ff3a71
Signed-off-by: Mark Yang <mark.yang@realtek.com>
